cmd/prysmctl/checkpoint/save.go

```diff
@@ -0,0 +1,75 @@
+package checkpoint
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"github.com/prysmaticlabs/prysm/api/client/beacon"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+var saveFlags = struct {
+	BeaconNodeHost string
+	Timeout        string
+}{}
+
+var saveCmd = &cli.Command{
+	Name:   "save",
+	Usage:  "Query for the current weak subjectivity period epoch, then download the corresponding state and block. To be used for checkpoint sync.",
+	Action: cliActionSave,
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:        "beacon-node-host",
+			Usage:       "host:port for beacon node connection",
+			Destination: &saveFlags.BeaconNodeHost,
+			Value:       "localhost:3500",
+		},
+		&cli.StringFlag{
```

Comment:
Have you considered cli.DurationFlag?

Reply:
I had not considered it, but I have now updated this code to use it, thanks!